packages/admin-stories/src/admin/form/AllFieldComponents.tsx

```diff
@@ -25,12 +25,8 @@ function Story() {
                                 <Field name="checkbox" label="FinalFormCheckbox" type="checkbox" component={FinalFormCheckbox} />
                                 <Field name="input" label="FinalFormInput" component={FinalFormInput} />
                                 <FieldContainer label="FinalFormRadio">
-                                    <Field name="radio" type="radio" value="foo">
-                                        {(props) => <FormControlLabel label="Foo" control={<FinalFormRadio {...props} />} />}
-                                    </Field>
-                                    <Field name="radio" type="radio" value="bar">
-                                        {(props) => <FormControlLabel label="Bar" control={<FinalFormRadio {...props} />} />}
-                                    </Field>
+                                    <Field name="radio" type="radio" value="foo" label="Foo" component={FinalFormRadio} />
```

Comment:
I like the previous API also.
What is the goal here? To abstract away the details of how field-labels are applied to form fields?

Is it now still possible to customise the FormControlLabel for checkboxes and radio? Or do we not need this anyway?

Reply:
The goal: Creating a checkbox or radio should work exactly the same as any other field, by just setting the component. 

Previously, by default, for every field, the label would be above the form-field. I don't think this should ever be the case for checkboxes and radios (it's confusing and looks weird). That's why I think this should always be the behaviour for these components, without having to create a custom `<FormControlLabel />` for every `<Field />`. 

The only case, where I think there should be a label above the checkbox/radio is, if there are multiple within the same category/group, but in this case the fields should be wrapped inside a `<FieldContainer label={...} />` anyway, and every checkbox/radio needs it's own label anyway to distinguish them from each other.

Reply:
![checkbox before after](https://user-images.githubusercontent.com/6264317/116870961-c7b7ce00-ac13-11eb-9651-82065e1d3a26.jpg)

Reply:
I would consider using an API similar to material-ui, as seen here https://material-ui.com/components/radio-buttons/#RadioButtonsGroup.tsx

of course, we need FinalForm `<Fields>` as well, so our usage will be different (we have a FinalFormRadio or something similar that does FinalForm integraion tough)

Reply:
> The goal: Creating a checkbox or radio should work exactly the same as any other field, by just setting the component.

while that is a noble goal, I think it won't work like that. Radios/Checkboxes are too different to other inputs.

Reply:
@nsams ok, I have created #370 so the inputs look good when copy-pasting form the stories. 

Should this be the permanent solution, or could we simplify this some other way?

Reply:
I think it is ok like that. But maybe someone has an idea on how to simplify?